scripts/demo/demo_benchmark_app.sh

```diff
@@ -211,15 +211,15 @@ cmake -DCMAKE_BUILD_TYPE=Release "$samples_path"
 make $NUM_THREADS benchmark_app
 
 # Step 5. Run samples
-printf "${dashes}"
+printf "%s" "${dashes}"
 printf "Run Inference Engine benchmark app\n\n"
 
 cd "$binaries_dir"
 
 cp -f "$ROOT_DIR/${model_name}.labels" "${ir_dir}/"
 
-print_and_run ./benchmark_app -d "$target" -i "$target_image_path" -m "${ir_dir}/${model_name}.xml" -pc ${sampleoptions}
+print_and_run ./benchmark_app -d "$target" -i "$target_image_path" -m "${ir_dir}/${model_name}.xml" -pc "${sampleoptions}"
```

Comment:
I don't think this will work correctly. It'll pass all options as one big option. `sampleoptions` should be changed to an array instead.